eval/judge.py

```
@@ -0,0 +1,264 @@
+        ]
+
+    source_lines = source_file.read_text().splitlines() if source_file.exists() else []
+    gt_summary = "\n".join(f"- [{g.id}] {g.title} ({g.severity})" for g in ground_truth)
+
+    models = ["claude-sonnet-4", "gemini-2.5-pro"]
+    verdicts = []
+
+    for f in findings:
+        start = max(0, f.line_start - 25)
+        end = min(len(source_lines), f.line_end + 25) if f.line_end > 0 else min(len(source_lines), 50)
+        excerpt = "\n".join(f"{i+start+1}: {l}" for i, l in enumerate(source_lines[start:end]))
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)